boards/nxp/frdm_ke17z512/board.cmake

```diff
@@ -0,0 +1,11 @@
+#
+# Copyright 2024 NXP
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+board_runner_args(jlink "--device=MKE17Z512xxx9" "--reset-after-load")
+board_runner_args(linkserver "--device=MKE17Z512xxx9:FRDM-KE17Z512")
+
+include(${ZEPHYR_BASE}/boards/common/jlink.board.cmake)
+include(${ZEPHYR_BASE}/boards/common/linkserver.board.cmake)
```

Comment:
nit, for consistency, all of our boards are defaulting to linkserver support, if you could switch the order of these include statements and update the board/doc/index stating its using linkserver support by default.